cf-java-logging-support-servlet/src/main/java/com/sap/hcp/cf/logging/servlet/filter/LogRemoteIPSettings.java

```diff
@@ -0,0 +1,45 @@
+package com.sap.hcp.cf.logging.servlet.filter;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.sap.hcp.cf.logging.common.helper.Environment;
+
+public class LogRemoteIPSettings {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(LogRemoteIPSettings.class);
+
+    private final String logRemoteIPSetting;
```

Comment:
Is there a specific reason why this is a String? Wouldn't it be sufficient if it were a boolean? I see that Environment has only one method returning a String, but that doesn't necessarily mean that the variable needs to be a String. Or does it?

Reply:
Has been changed to boolean.